math/nickle/Portfile

```diff
@@ -17,14 +18,41 @@ long_description    Nickle is a programming language based prototyping environme
                     have been made differently, and a very few features are simply missing.
 homepage            https://nickle.org
 master_sites        ${homepage}/release/
-checksums           rmd160  1bc9c33a56fb1083b27cc68a047aa53b4a728ec5 \
-                    sha256  e2d6434d1216aeec552932828f9aed52f326bd12d8e06cae0f0de4a119d40a3f \
-                    size    2553393
+use_xz              yes
+checksums           rmd160  e8a76e9a54579d3510bb57547f3c042b73c637f7 \
+                    sha256  f802e6c229f6520887083acacaf653e01cdecec3ffad3f9be3bee621e8d747d7 \
+                    size    2189344
 
+# Nickle switched to generating tables for tests,
+# and its script uses invalid syntax, which breaks
+# math test. autotools buils cannot generate tables:
+# fmt: illegal option -- -
+# meson build generates broken tables:
+# 11/22 math.5c FAIL 1.18s exit status 1
+# math-tables.5c: syntax error before "exp"
+# It is not enough to replace invalid --width with -width
+# or alternatives, since lines with . are not handled
+# correctly anyway. So unfortunately, we need to depend
+# on coreutils here, which can handle linuxisms.
+patchfiles-append   patch-fix-broken-tests-syntax.diff
```

Comment:
please open an upstream issue for them to resolve this; it's not something MacPorts wants to patch forever

Reply:
I did not find an upstream repo where I can submit a patch.

This is my port, by the way. Please let me maintain it. And I am pretty sure that no one gonna touch it for years, If I do not do that, so the problem is not with patching forever here.

Reply:
(1) latest upstream version is `2.101` ; please check if some stuff has been resolved there already.  (2) Upstream appears active so there must be a way to reach them.